MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+recursive-include perun *.pyi
+include perun/py.typed
```

Comment:
What is this `py.typed` file? It is some stub? Can we add some comment why it is here and why it is empty? Can it contain like comment `# Empty file needed for lazy_loading`?

Reply:
This is a requirement of [PEP 561](https://peps.python.org/pep-0561/) and described also in [mypy documentation](https://mypy.readthedocs.io/en/stable/installed_packages.html#creating-pep-561-compatible-packages). Packages that distribute both runtime and type stub files (`.pyi` files) need to contain a `py.typed` file as well to indicate support for type hints. The MANIFEST.in file is then needed for sdist distribution to include the `.pyi` and `py.typed` files. As the PEP does not specify what the `py.typed` files should contain and it is easy enough to find an explanation for the file online, I'd just keep it empty.